test/TestUtilities/Test.Utility/Signing/TestFolderNames.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Test.Utility.Signing
+{
+    /// <summary>
+    /// Folder names ensure uniqueness in path when signed packages are stored on disk for verification in later steps
+    /// </summary>
+    public enum FolderNames
+    {
+        One,
```

Comment:
these enum values didn't give me a clue about the purpose...the summary helps.  Perhaps better enum name or value names could make it more obvious??